docs/release-notes/NuGet-5.6.md

```diff
@@ -0,0 +1,65 @@
+# 5.6 Release Notes
+
+[Full Changelog]("")
+
+[Issues List](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=5e3b2080c4b30708e48bf9f3)
+
+**DCR:**
+
+* Support pre-release packages with floating versions. Version="*-*" and similar. - [#912](https://github.com/NuGet/Home/issues/912)
+
+* add support to `NuGet.exe update` for -DependencyVersion parameter, like install command - [#7694](https://github.com/NuGet/Home/issues/7694)
+
+* Default verbosity should not report each project's noop restore. - [#8792](https://github.com/NuGet/Home/issues/8792)
+
+* Sort packages by ID in the Updates tab of the Package Manager UI - [#9278](https://github.com/NuGet/Home/issues/9278)
+
+* net5.0 tfm - initial support - [#9584](https://github.com/NuGet/Home/issues/9584)
+
+**Bug:**
+
+* dotnet nuget push - Missing value for option - [#4864](https://github.com/NuGet/Home/issues/4864)

```

Comment:
This issue is still open.

Reply:
was in 5.6 release. just fixed that. will add that as 1 more bug to release note tool.